lib/MusicBrainz/Server/Controller/Area.pm

```diff
@@ -256,6 +256,29 @@ sub places : Chained('load')
             ],
         }),
     );
+
+    my %props = (
+        area        => $c->stash->{area},
+        mapDataArgs => $c->json->encode({
+            places => [
+                map {
+                    my $json = $_->TO_JSON;
+                    # These arguments aren't needed at all to render the map,
+                    # and only increase the page size.
+                    delete @{$json}{qw(annotation unaccented_name)};
+                    $json;
+                } grep { $_->coordinates } @$places
+            ],
+        }),
+        places      => $places,
```

Comment:
Is it still necessary to stash `places` (and `map_data_args`) above?

Reply:
Oh. Duh. No, it really isn't.